src/FileHelper.php

```diff
@@ -51,9 +51,7 @@ public static function createDirectory(string $path, int $mode = 0775): bool
         $path = static::normalizePath($path);
 
         try {
-            if (!mkdir($path, $mode, true) && !is_dir($path)) {
-                return false;
-            }
+            return \mkdir($path, $mode, \true) && \is_dir($path);
```

Comment:
This change is incorrect - right now `return static::chmod($path, $mode)` is never called.

/cc @samdark 

BTW: Why `\true`? `true` is a keyword, it is pointless to prefix it by `\`.

Reply:
You're right. Thanks for noticing it.